src/flakeref/fr_type.rs

```diff
@@ -74,7 +82,12 @@ impl FlakeRefType {
         })(input)
     }
     pub fn parse_explicit_file_scheme(input: &str) -> IResult<&str, &Path> {
-        let (rest, _) = alt((tag("path:"), tag("file://"), tag("file+file://")))(input)?;
+        let (rest, _) = alt((
+            tag("file://"),
+            tag("file+file://"),
+            tag("file:"),
+            tag("file+file:"),
+        ))(input)?;
```

Comment:
Before, the string leading with "path:" would resolve to a `FlakeRef::Resource{...` with the `File` resource type. now it resolves to a `FlakeRef::Path{....`